contents/docs/tutorials/actions.md

```diff
@@ -0,0 +1,196 @@
+---
+title: Going Beyond Autocapture
```

Comment:
The concept of the title is valid, but as a tutorial name it isn't self explanatory. I feel like this is more of a 'Complete guide to event tracking', do you agree?

Reply:
Yeah great point - the title tells the reader very little. 

Will change to ''Complete guide to event tracking' or something like that. Don't need PostHog in the title